test_fixtures/roberta_model/vocabulary/d_tags.txt

```diff
@@ -0,0 +1,4 @@
+CORRECT
```

Comment:
Are the files in this directory the same as the ones in `data/output_vocabulary`?

Reply:
yep they are! Should I make a commit to delete the files from data/output_vocabulary?

Reply:
Yeah, I think so. We need to do an overhaul of the README anyway, so we can just point to this directory for the `vocab_path`.

Reply:
Actually, I think Damien and I can do this in the next PR. The vocab path will need to be changed in all the tests.